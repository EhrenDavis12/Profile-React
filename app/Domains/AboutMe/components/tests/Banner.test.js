import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import BannerSection from '../Banner';

describe('<BannerSection />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(
      <BannerSection>
        <div>Hello Wold</div>
      </BannerSection>,
    );
    expect(renderedComponent).toMatchSnapshot();
  });
});
