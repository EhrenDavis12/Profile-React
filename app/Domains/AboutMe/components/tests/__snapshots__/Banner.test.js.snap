// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BannerSection /> should render the page 1`] = `
<Banner__Background>
  <div>
    Hello Wold
  </div>
</Banner__Background>
`;
